db/seeds.rb

```diff
@@ -1,7 +1,23 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
```

Comment:
Why don't we add faker to the Gemfile so you don't need to require and include it? `gem 'faker'` in the development group, and then run bundle install. You can remove these two lines at that point. The advantage is that Faker is then available in any other files where Rails is autoloading things.